minidec/main.rs

```diff
@@ -59,6 +67,7 @@ fn main() {
         //println!("Local Variable info: {:?}", rfn.locals);
         let mut writer: IRWriter = Default::default();
         writer.emit_il(Some(rfn.name.clone()), &rfn.ssa, &mut ff);
+        writer.emit_il(Some(rfn.name.clone()), &rfn.ssa, &mut ffm);
```

Comment:
I already tried.
Rust complained: `error: the trait bound 'std::string::String: std::io::Write' is not satisfied [E0277]`
I was going to implement the `std::io::Write` trait for `std::string::String`. Would this be the way to go? (Haven't found the time yet.)